logstash-core-event-java/src/main/java/com/logstash/Event.java

```diff
@@ -253,16 +297,16 @@ private Timestamp initTimestamp(Object o) {
             } else if (o instanceof String) {
                 // second most frequent
                 return new Timestamp((String) o);
-            } else if (o instanceof JrubyTimestampExtLibrary.RubyTimestamp) {
-                return new Timestamp(((JrubyTimestampExtLibrary.RubyTimestamp) o).getTimestamp());
+//            } else if (o instanceof JrubyTimestampExtLibrary.RubyTimestamp) {
+//                return new Timestamp(((JrubyTimestampExtLibrary.RubyTimestamp) o).getTimestamp());
```

Comment:
Could not get JMH to compile JRuby without making it a compile dependency for the normal Jar.

I will try again.